app/helpers/dynamic_errors_helper.rb

```diff
@@ -0,0 +1,55 @@
+module DynamicErrorsHelper
```

Comment:
If we are really adding this to app folder, we should compact it to exactly the code we use and add tests, which I think is better solution than copying to lib.